tck/features/expressions/map/Map4.feature

```diff
@@ -35,18 +35,28 @@ Feature: Map4 - Field existence check
     When executing query:
       """
       WITH <map> AS map
-      RETURN exists(map.name) AS result
+      RETURN exists(map.<key>) AS result
       """
     Then the result should be, in any order:
       | result   |
       | <result> |
     And no side effects
 
     Examples:
-      | map                             | result |
-      | {name: 'Mats', name2: 'Pontus'} | true   |
-      | {name: null}                    | false  |
-      | {notName: 0, notName2: null}    | false  |
+      | map                                 | key     | result |
+      | {name: 'Mats', name2: 'Pontus'}     | name    | true   |
+      | {name: 'Mats', name2: 'Pontus'}     | name2   | true   |
+      | {name: null}                        | name    | true   |
+      | {name: null, name2: 'Pontus'}       | name    | true   |
+      | {name: null, name2: null}           | name    | true   |
+      | {name: null, name2: null}           | name2   | true   |
+      | {name: 'Pontus', name2: null}       | name2   | true   |
```

Comment:
this looks identical to line 50; one null value and one non-null, and then we reach for the key with null value.

Reply:
Again, the order of field does not matter. If you say, these scenarios are identical, you implicitly assume an implementation follows the rule that with field order does not matter. Go tests do not make such assumptions.

Reply:
I would've thought it was another test's responsibility to check that field order in maps have no meaning, but I'm happy to keep the scenario given your reasoning.